db/migrate/20170805001646_drop_delayed_jobs.rb

```diff
@@ -0,0 +1,9 @@
+class DropDelayedJobs < ActiveRecord::Migration[5.1]
+  def up
+    drop_table "delayed_jobs"
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

Comment:
Does this really need to be irreversible?  The contents of the delayed_jobs table are ephemeral anyway...  And having irreversible migrations makes it *really* hard to test and upgrade the database when needed.